packages/components/src/Stack/Stack.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Stack, Text } from '@marigold/components';
+
+test('supports default space prop', () => {
+  render(
+    <Stack title="stack">
+      <Text>stack</Text>
+    </Stack>
+  );
+  const stack = screen.getByTitle(/stack/);
+
+  expect(stack).toHaveStyle(`padding: 0px`);
```

Comment:
Shouldn't that be `padding-top`?

Reply:
Could be both. In the first box I rendered its padding and in the box from the map() its pt. I think this is enough?

Reply:
But you are testing the inner components. Shouldn't they only have top padding

Reply:
No in this case i am testing the outer component.
"getByTitle"